test/client/fixtures/sandbox/shadow-ui-test.js

```diff
@@ -438,6 +439,117 @@ test('querySelectorAll', function () {
     strictEqual(elems[0].id, 'pageElem');
 });
 
+module('ui stylesheet');
+
+if (isWebKit) {
```

Comment:
It's not impossible, because we used browser detection [in source code](https://github.com/LavrovArtem/testcafe-hammerhead/commit/b64f0f7ec2e83c06cd7b1db599230cef4f089016?diff=unified#diff-b69e00cc96d6609a3e7336ffc385eae6L173)